doc/source/concepts/concepts.rst

```diff
@@ -627,6 +627,21 @@ The following will describe an Rsync file transfer operation over SSH to downloa
     ct.fs.FileTransfer(from_remote_file, to_local_file, strategy=strategy)
 
 
+S3
+~~~~~~~~~~~~
+.. warning:: S3 must be installed on an electron’s backend execution environment. It can be installed using :code:`pip install boto3`
```

Comment:
```suggestion
.. warning:: AWS Python SDK must be installed on an electron’s backend execution environment. It can be installed using :code:`pip install boto3`
```